snaps/reference/rpc-api.md

```diff
@@ -984,6 +984,11 @@ console.log(result); // In this example, the result is a boolean.
 
 ### wallet_invokeSnap
 
+:::caution
+This method is deprecated.
+Use [`wallet_requestSnaps`](#wallet_requestsnaps) instead.
```

Comment:
Wait this is wrong. `wallet_invokeSnap` is not deprecated. It is the correct way to communicate with a Snap after connecting to it.